index.html

```diff
@@ -36,6 +36,7 @@
             <div id="install">
                 <ul class="commands commands-tooltip">
                     <li>cd /boot/EFI/refind/</li>
+                    <li>mkdir -p themes && cd themes</li>
                     <li>git clone {{site.github.repository_url}}</li>
```

Comment:
Maybe instead of `cd`ing before hand just clone into `themes/{{site.gihtub.repository_url}}`